pkg/plugin/plugin_init.go

```diff
@@ -94,7 +94,10 @@ func InitPlugin(pluginName string, newEventHandler EventHandlerCreator, newServe
 		100)
 
 	handler := newEventHandler(githubClient, *pluginBotName)
+
+	server.RegisterMetrics(logger)
 	pluginServer := newServer(webhookSecret, handler)
+	pluginServer.Metrics = server.NewMetrics()
```

Comment:
this looks a bit weird - that it first registers the metrics and then calls the constructor. I think that this should be part of one step (method)